recipes/fontconfig/fontconfig.inc

```diff
@@ -10,7 +10,15 @@ SRC_URI = "http://fontconfig.org/release/${PN}-${PV}.tar.gz"
 
 S = "${SRCDIR}/${PN}-${PV}"
 
-EXTRA_OECONF = "--disable-docs --with-arch=${HOST_ARCH} --with-cache-dir=${localstatedir}/lib/${PN}"
+EXTRA_OECONF = "--disable-docs --with-arch=${HOST_ARCH}"
+
+RECIPE_FLAGS += "fontconfig_dir_cache"
+DEFAULT_USE_fontconfig_dir_cache = "--with-cache-dir=${localstatedir}/lib/${PN}"
```

Comment:
In an ideal world, perhaps, but the existing downstream MACHINE_USE_ contain that, and I'd like to avoid having One More Thing to worry about when switching back and forth between old and new meta/base.

Reply:
OK, I'll do it with other names (cache_dir instead of dir_cache, which better matches the config option anyway), then I can set both variants appropriately downstream.